dts/msm8974/msm8974pro-ac-pm8941-mtp.dts

```diff
@@ -16,7 +16,7 @@
 	oneplus-bacon {
 		model = "OnePlus One";
 		compatible = "oneplus,bacon", "qcom,msm8974", "lk2nd,device";
-		lk2nd,match-cmdline = "* androidboot.hardware=bacon *";
+		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_jdi_1080p_cmd *";
```

Comment:
Is there only one panel known to be used in the field? If no, maybe like this?

https://github.com/msm8916-mainline/lk2nd/blob/master/dts/msm8916/msm8916-qrd-skut1.dts#L15

Reply:
To my knowledge, yes. If we find more in the future I'll expand this for sure (and then we also need to use the panel selection of lk2nd + kernel support for the new panel)